src/library/coverart.h

```diff
@@ -7,7 +7,8 @@
 #include <QtDebug>
 #include <QtGlobal>
 
-struct CoverInfo {
+class CoverInfoRelative {
```

Comment:
Do we really need a class hierarchy with base/inner classes that don't have any virtual functions? It would be safer to use composition instead of inheritance.